Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Repository.rst

```diff
@@ -152,87 +152,32 @@ Typo3QuerySettings and localization
 
 Extbase renders the translated records in the same way as TypoScript rendering.
 
-..  versionchanged:: 11.0
-    Setting :php:`Typo3QuerySettings->languageMode`  was deprecated and does
-    **not** influence how Extbase queries records.
+..  versionchanged:: 12.0
+    The Extbase query settings rely on
```

Comment:
We should mention here that the old methods still work in v12, but are deprecated.